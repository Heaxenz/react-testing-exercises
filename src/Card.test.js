import { render } from "@testing-library/react";
import Card from "./Card";

it('renders the Card component', () => {
    render(<Card />)
})

it('Should create a card with the given parameters', () => {
    const {asFragment} = render(<Card caption='Hello' src="http://localhost:3000/static/media/image1.08e1519e765278b10548.jpg"/>)
    expect(asFragment()).toMatchSnapshot();
})