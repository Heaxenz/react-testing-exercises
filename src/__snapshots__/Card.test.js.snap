// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should create a card with the given parameters 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    >
      Hello
    </h4>
    <img
      alt="Hello"
      class="Card-image"
      src="http://localhost:3000/static/media/image1.08e1519e765278b10548.jpg"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
